### PR TITLE
Handle conversion of boolean query parameters with a value of "false"

### PR DESCRIPTION
**Pull request by csrwng**

### CODE REVIEW

**Comment by csrwng** [comment]
@smarterclayton ptal

File: pkg/runtime/conversion.go [hunk 1]
```diff
@@ -74,6 +74,8 @@ func convertStringSliceToBool(input *[]string, out *bool, s conversion.Scope) er
 	switch strings.ToLower((*input)[0]) {
 	case "true", "1":
 		*out = true
+	case "false", "0":
+		*out = false
 	default:
 		*out = true
```
Comment (smarterclayton) [inline pkg/runtime/conversion.go]: Should probably omit the `case "true", "1"` now - this is confusing at first sight.  Add a comment that only false and 0 resolve to false, all other non-zero length values are true.

**Comment by csrwng** [comment]
updated and added more tests

**Comment by smarterclayton** [comment]
LGTM, travis flake rerunning

**Pull request merged**